.circleci/fetch_merge_commit.sh

```diff
@@ -25,17 +25,17 @@ err=0
 GREEN() { echo -e "\n\033[0;32m$1\033[0m"; }
 RED() { echo -e "\n\033[0;31m$1\033[0m"; }
 
-# CIRCLE_PULL_REQUEST is present for PR builds, and absent for push builds.
-if [[ -z "$CIRCLE_PULL_REQUEST" ]]; then
+# CIRCLE_PR_NUMBER is present for PR builds, and absent for push builds.
+if [[ -z "$CIRCLE_PR_NUMBER" ]]; then
```

Comment:
This won't work for branches on `ampproject/amphtml` because of what's written on line 40.

https://github.com/ampproject/amphtml/blob/bb44fbd992113dd1ac4156610a33850413b3dabd/.circleci/fetch_merge_commit.sh#L40-L42

Reply:
I'm certain this is because the branch gets created first, and the PR gets created a few seconds later. Meanwhile, CircleCI kicks off a build as soon as the branch is created.

Reply:
And the more I think about this, there's probably nothing we can do except run all tests when someone directly creates a branch on `ampproject/amphtml`. Let's merge #32412 with my suggestions committed to it.

Reply:
Never mind, I figured out a way to fix this. Follow up in https://github.com/ampproject/amphtml/pull/32417.